include/drivers/usbc/usbc_tc.h

```diff
@@ -0,0 +1,435 @@
+/* usbc_tc.h - USB Type-C Cable and Connector */
+
+/*
+ * Copyright 2022 The Chromium OS Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief USBC Type-C Cable and Connector API used for USBC drivers
+ *
+ * The information in this file was taken from the USB Type-C
+ * Cable and Connector Specification Release 2.1
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_USBC_USBC_TC_H_
+#define ZEPHYR_INCLUDE_DRIVERS_USBC_USBC_TC_H_
+
+/**
+ * @brief USB Type-C
+ * @defgroup _usb_type_c USB Type-C
+ * @{
+ */
+
+#include <zephyr/types.h>
+#include <device.h>
```

Comment:
nit: this header doesn't seem to use any of the device APIs

Reply:
Done